packages/curve-ui-kit/src/shared/ui/DataTable/DataTable.d.ts

```diff
@@ -12,6 +12,7 @@ declare module '@tanstack/react-table' {
     variant?: TypographyVariantKey
     borderRight?: boolean
     hideZero?: boolean
+    noPadding?: boolean
```

Comment:
This is no longer being used, is it?

Reply:
yes sorry, I was supposed to remove this